src/Service.Tests/dab-config.CosmosDb_NoSql.json

```diff
@@ -105,48 +84,28 @@
       "actions": [
         {
           "action": "create",
-          "fields": null,
-          "policy": {
-            "request": null,
-            "database": null
-          }
+          "policy": {}
         },
         {
           "action": "read",
-          "fields": null,
-          "policy": {
-            "request": null,
-            "database": null
-          }
+          "policy": {}
```

Comment:
policy is not supported for cosmosdb_nosql at this point. Can we skip writing the policy as well? Can we define it nullable so it doesnt write to the file?

Reply:
We'd be better off making `Policy` on `EntityAction` a nullable property so we can null it out, rather than nulling its properties.

I'll see what the complexity/cascading impacts of that is.

Reply:
Made `Policy` nullable and that has cleaned up the config file even more.

Reply:
If it was made nullable, why do we write the empty object to the config file. Cant we skip writing it altogether?

Reply:
Looks like I've missed a place where it's being initialized still. I'll clean that up.